packages/@okta/vuepress-site/docs/concepts/saml/index.md

```diff
@@ -37,7 +37,7 @@ SAML is mostly used as a web-based authentication mechanism inasmuch as it relie
 
 We are now ready to introduce some common SAML terms. We will go into the technical details of these later, but it is important to understand the high-level concept during the planning stage.
 
-- **A Service Provider (SP)** is the entity providing the service, typically in the form of an application
+- **A Service Provider (SP)** is the entity providing the service, typically in the form of an application.
```

Comment:
@susanharper-okta, what's the latest and greatest style guideline for period/no-period here?

Reply:
My issue is that "A" and "An" (below) should not be in bold. I mean, according to our style guide none of it should be in bold, but at least not "a/an".

Reply:
@motikagan-okta @susanharper-okta , as an fyi, I noticed both styles in bulleted lists in several pages (i.e., no period in some lists, periods in others). In this case, I simply opted for consistency. (And mainly testing out my access :o)

Reply:
@brianduffield-okta - let's go with the following:
`- A **Service Provider (SP)** is the entity providing the service, typically in the form of an application.`